### DISCUSSION TITLE
What is the purpose of useQuery and why does the documentation not highlight its purpose?

### DISCUSSION THREAD

**Discussion by NathanielPather** [TanStack/query#6379] [Q&A] [answered]
The documentation should briefly explain what useQuery does.
Currently the documentation doesn't provide a brief overview of what the purpose of the function is.

![image](https://github.com/TanStack/query/assets/12907920/b3c3a705-4dbb-4110-ad5a-030b1d2e88fa)

Should it?

**Comment by TkDodo** [comment, marked as answer]
That's the API documentation. Have you read the guides?

**Reply by NathanielPather** [reply]
What's the difference between the API documentation and the guides? I assumed the documentation should outline the purpose, not to mention the code should have some auto generating documentation for me to click and see in my IDE?

**Reply by TkDodo** [reply]
api documentation tells you which arguments you can pass to which function; The guides tell you how to use the library.
